### PR TITLE
add css grid

### PR DESCRIPTION
**Pull request by isnotafunction**
This PR:
- is just a quick fix for desktop appearance done using css grid, given that we designed for mobile first
- removes some of the section tags where they do not have the purpose of thematically grouping the content (i.e. when the only content they have is one heading)
- removes the main tag > wasn't sure if it makes sense to use it given the layout of our app but I might be wrong.
- we need to look into profile.js, there was an error 'can't read [0] of undefined' to get the initial out of the username of the logged in user, I quickly changed it to using the params just to make it work, but we need to fix this.

This PR doesn't fix: 
- inconsistencies in the whole of CSS re units, repetition etc.
- accessibility 
- anything else

### CODE REVIEW

File: src/views/layouts/main.hbs [hunk 1]
```diff
@@ -1,18 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
 
- {{>head}}
+{{>head}}
 
 <body>
-{{#if home}}
+  {{#if home}} {{else}} {{> header}} {{/if}} {{{ body }}}
```
[the remark this answers is not in the document]
Comment (isnotafunction) [reply src/views/layouts/main.hbs]: I removed the main and sections that are not necessary in terms of styling especially in places where they only have a header element but no other related content

File: src/views/error.hbs [hunk 2]
```diff
@@ -1,3 +1 @@
-<section>
-  <h1>{{statusCode}} {{errorMessage}}</h1>
-</section>
\ No newline at end of file
+<h1 class="title">{{statusCode}} {{errorMessage}}</h1>
```
[the remark this answers is not in the document]
Comment (isnotafunction) [reply src/views/error.hbs]: See below

File: public/index.js [hunk 3]
```diff
@@ -8,64 +7,57 @@ const randomGift = document.getElementById('link__randomGift');
...
  * fetch API content from server
  */
-const fetchContent = (url) => {
+const fetchContent = url => {
   fetch(url)
     .then(response => {
       return response.json();
     })
-    .then((data) => {
+    .then(data => {
```
[the remark this answers is not in the document]
Comment (isnotafunction) [reply public/index.js]: I'll adjust the settings!

File: src/app.js [hunk 4]
```diff
@@ -25,13 +25,12 @@ app.engine(
     extname: 'hbs',
     layoutsDir: path.join(__dirname, 'views', 'layouts'),
     partialsDir: path.join(__dirname, 'views', 'partials'),
-    defaultLayout: 'main',
-  }),
+    defaultLayout: 'main'
```
[the remark this answers is not in the document]
Comment (isnotafunction) [reply src/app.js]: see above

File: src/views/layouts/main.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Parissai) [reply src/views/layouts/main.hbs]: We need them accessibility wise. Without them, if you use outline extension of chrome, you will see that we have an untitled body. We just need to give the section role attr.

**Pull request closed without merging**